proto/rpc/v1/auth.proto

```diff
@@ -9,9 +9,11 @@ import "google/api/annotations.proto";
 // authentication. Its sole Authenticate method should be used prior to any
 // other services that a gRPC server has to offer.
 service AuthService {
-	// Authenticate attempts to authenticate the caller. The resulting
-	// response contains an access token that should be used for future
-	// requests.
+	// Authenticate attempts to authenticate the caller claiming to be
```

Comment:
Hopefully this clears up some questions for future readers.

Reply:
I should have never called these entities but oh well

Reply:
[nit] need some generic name anyways to support multiple types (users, robots, service_account, etc)